rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -0,0 +1,179 @@
+from werkzeug.exceptions import BadRequest
+
+from .participant_summary_dao import ParticipantSummaryDao
+from model.participant_summary import ParticipantSummary
+from participant_enums import EnrollmentStatus
+
+class ParticipantCountsOverTimeService(ParticipantSummaryDao):
+
+  def __init__(self):
+    super(ParticipantSummaryDao, self).__init__(ParticipantSummary)
+
+  def get_filtered_results(self, start_date, end_date, filters, stratification='ENROLLMENT_STATUS'):
```

Comment:
You may want to break this method up to handle each stratification with a separate helper method... each query is pretty complicated on its own.

Reply:
Good idea, done.